README.md

```diff
@@ -15,7 +15,7 @@ There are two available classes with a different behaviour:
 ## MutableArray
 
 Each functional method operate on the same array and return the same instance
-(DO NOT create a new instance) except only few methods starts with `create` prefix.
+(**DO NOT** create a new instance) except only few methods starts with `create` prefix.
```

Comment:
This is a small consistency fix, look at this section:
https://github.com/bocharsky-bw/Arrayzy#createclone